### PR TITLE
add: add new blog post

### PR DESCRIPTION
**Pull request by dreyfus92**
I just wrote a post about the temporal proposal of tc39

### CODE REVIEW

File: src/content/blog/temporal-stage3-proposal.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+tags: ['javascript', 'proposal', 'tc39']
+author: paul-valladares
+canonicalUrl: 'https://www.paulvall.dev/posts/temporal-proposal'
+excerpt:
+  'Taking a look at the Temporal proposal, which aims to modernize date and time handling in JavaScript.'
+---
+
+## Introduction
+
+The handling of dates in ECMAScript has long been a source of frustration for developers. In response to this, the Temporal proposal was born. Temporal is a global object, similar to Math, that introduces a modern date and time API to the ECMAScript language. To dive deeper into the challenges associated with the existing Date object and the motivations behind Temporal, take a look at this informative article: Fixing JavaScript Date.
```
Comment (nikolovlazar) [inline src/content/blog/temporal-stage3-proposal.md:15]: You're missing the `Fixing JavaScript Date` article link.

File: src/content/blog/temporal-stage3-proposal.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+**User-Friendly APIs:** Temporal provides easy-to-use APIs for various date and time computations, making complex operations more accessible.
+
+**Time Zone Support:** It boasts first-class support for all time zones, including daylight-saving time (DST) adjustments, ensuring accurate and reliable arithmetic across different regions.
+
+**Immutability:** Temporal enforces immutability, meaning that once a date or time object is created, it cannot be altered. This helps prevent bugs stemming from unintended modifications.
+
+**Strict String Parsing:** Temporal allows you to parse date and time strings in a strictly specified format, ensuring consistency in data handling.
+
+**Non-Gregorian Calendars:** Temporal even goes beyond the Gregorian calendar, accommodating different calendar systems, broadening its usability.
```
Comment (nikolovlazar) [inline src/content/blog/temporal-stage3-proposal.md:29]: Turn these into a list:

```markdown
- **User-Friendly APIs:** ...
- **Time Zone Support:** ...
- **Immutability:** ...
- **Strict String Parsing:** ...
- **Non-Gregorian Calendars:** ...
```

File: src/content/blog/temporal-stage3-proposal.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+Temporal introduces a powerful feature known as Temporal.Now, offering real-time insights into the current system time and time zone. Developers can leverage this feature to access:
+
+`Temporal.Now.instant()`: Retrieve the current system exact time.
+`Temporal.Now.timeZoneId()`: Obtain the current system time zone.
+`Temporal.Now.zonedDateTime(calendar)`: Fetch the current date and wall-clock time in the system time zone and a specified calendar.
+`Temporal.Now.zonedDateTimeISO()`: Access the current date and wall-clock time in the system time zone using the ISO-8601 calendar.
+`Temporal.Now.plainDate(calendar)`: Obtain the current date in the system time zone and a specified calendar.
+`Temporal.Now.plainDateISO()`: Retrieve the current date in the system time zone using the ISO-8601 calendar.
+`Temporal.Now.plainTimeISO()`: Retrieve the current wall-clock time in the system time zone using the ISO-8601 calendar.
+`Temporal.Now.plainDateTime(calendar)`: Retrieve the current system date/time in the system time zone. Note that the resulting object does not remember its time zone, making it unsuitable for deriving other values in time zones that observe Daylight Saving Time (DST).
```
Comment (nikolovlazar) [inline src/content/blog/temporal-stage3-proposal.md:54]: You can put these into a list as well. I think they're going to read better.

**Comment by dreyfus92** [comment]
Thank you for the review @nikolovlazar

**Review by nikolovlazar — APPROVED** [review]
Looking good! Thanks for contributing this article @dreyfus92 💖

**Pull request merged**